examples/rvv-memcpy/solution.s

```diff
@@ -0,0 +1,16 @@
+.globl rvv_memcpy
```

Comment:
Can you add 

```yaml
 - name: Run rvv-memcpy example 
   run: bin/run rvv-memcpy
```

under

https://github.com/codewars/riscv/blob/0679125cdd8326acf398c617909a681e06191c71/.github/workflows/ci.yml#L45-L46

to run this example in CI?

Reply:
Done. Separate commits for now, will squash if needed after we decide on how to deal with qemu on host.